python/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -341,6 +341,10 @@ class HDBSCAN(UniversalBase, ClusterMixin, CMajorInputTagMixin):
     (and often will) lead to some points being assigned different
     cluster labels between the two implementations.
 
+    This estimator supports cuML's experimental device selection capabilities.
+    It can be configured to run on either the CPU or the GPU.
+    To learn more, please see :ref:`device-selection`.
+
```

Comment:
Should we maybe automatically add this comment via a common decorator? Something like `@device_interop_enabled` ? This would also allow us to (maybe at a later stage) set some flag that helps with testing.